GateWatcher/aioniq_ecs/_meta/fields.yml

```diff
@@ -0,0 +1,1299 @@
+  name: gatewatcher.sip.uri
+  type: text
+
+gatewatcher.sip.version:
+  description: sip version field
+  name: gatewatcher.sip.version
+  type: text
+
+gatewatcher.smb.command:
+  description: smb command field
+  name: gatewatcher.smb.command
+  type: text
+
+gatewatcher.smb.dialect:
+  description: smb dialect field
+  name: gatewatcher.smb.dialect
+  type: text
+
+gatewatcher.smb.id:
+  description: smb id field
+  name: gatewatcher.smb.id
+  type: number
+
+gatewatcher.smb.session_id:
+  description: smb session id field
+  name: gatewatcher.smb.session_id
+  type: number
+
+gatewatcher.smb.status:
+  description: smb status field
+  name: gatewatcher.smb.status
+  type: text
+
+gatewatcher.smb.status_code:
+  description: smb status_code field
+  name: gatewatcher.smb.status_code
+  type: text
+
+gatewatcher.smb.tree_id:
+  description: smb tree_id field
+  name: gatewatcher.smb.tree_id
+  type: number
+
+gatewatcher.smtp.helo:
+  description: smtp helo field
+  name: gatewatcher.smtp.helo
+  type: text
+
+gatewatcher.smtp.mail_from:
+  description: smtp mail from field
+  name: gatewatcher.smtp.mail_from
+  type: text
+
+gatewatcher.smtp.rcpt_to:
+  description: smtp recipients field
+  name: gatewatcher.smtp.rcpt_to
+  type: text
+
+gatewatcher.snmp.community:
+  description: snmp community field
+  name: gatewatcher.snmp.community
+  type: text
+
+gatewatcher.snmp.pdu_type:
+  description: snmp pdu_type field
+  name: gatewatcher.snmp.pdu_type
+  type: text
+
+gatewatcher.snmp.vars:
+  description: snmp vars field
+  name: gatewatcher.snmp.vars
+  type: text
+
+gatewatcher.snmp.version:
+  description: snmp version field
+  name: gatewatcher.snmp.version
+  type: number
+
+gatewatcher.ssh.client.hassh:
+  description: ssh client hassh field
+  name: gatewatcher.ssh.client.hassh
+  type: text
+
+gatewatcher.ssh.client.proto_version:
+  description: ssh client proto_version field
+  name: gatewatcher.ssh.client.proto_version
+  type: text
+
+gatewatcher.ssh.client.software_version:
+  description: ssh client software_version field
+  name: gatewatcher.ssh.client.software_version
+  type: text
+
+gatewatcher.ssh.server.hassh:
+  description: ssh server hassh field
+  name: gatewatcher.ssh.server.hassh
+  type: text
+
+gatewatcher.ssh.server.proto_version:
+  description: ssh server proto_version field
+  name: gatewatcher.ssh.server.proto_version
+  type: text
+
+gatewatcher.ssh.server.software_version:
+  description: ssh server software_version field
+  name: gatewatcher.ssh.server.software_version
+  type: text
+
+gatewatcher.syslog.message:
+  description: syslog message field
+  name: gatewatcher.syslog.message
+  type: text
+
+gatewatcher.tftp.file:
+  description: tftp file field
+  name: gatewatcher.tftp.file
+  type: text
+
+gatewatcher.tftp.mode:
+  description: tftp mode field
+  name: gatewatcher.tftp.mode
+  type: text
+
+gatewatcher.tftp.packet:
+  description: tftp packet field
+  name: gatewatcher.tftp.packet
+  type: text
```

Comment:
In the legacy parser, we have the [`gatewatcher.tftp` field](https://github.com/SEKOIA-IO/intake-formats/blob/main/GateWatcher/aioniq/_meta/fields.yml#L376-L380) declared as a `text`.
Having a text field in the legacy format and a structured one in the new format will lead to collision in the indexation engine (and the events will be not indexed).
Two solutions:
- Fix the name of the field in the legacy format (I recommend this option).
- Fix the name of the field in this format

Reply:
Proposal fix in PR #1276